### DISCUSSION TITLE
How does saleor persist data?

### DISCUSSION THREAD

**Discussion by e-zer0** [saleor/saleor#8569] [General]
I'm new to Saleor and I'm examining the platform code.  Is the platform code using PostgreSQL to persist the data?  How is graphql communicating with the database?  Basically, what part of the saleor platform code can I examine to see what's going on?

**Comment by karolkielecki** [comment]
@e-zer0 Hi! GraphQL (query / mutation) -> graphene analyzes the request -> ORM code inside resolver or mutation hits the database (using Django, DB settings specified in settings.py)

**Reply by pmalawsk13** [reply]
Hi @karolkielecki 
do you know if any database model exist for Saleor DB?
